### DISCUSSION TITLE
Been waiting for 5 days for benefits, but have not received it. What should I do?

### DISCUSSION THREAD

**Discussion by Ronak1Nambiar** [community/community#135237] [GitHub Education]
### Hi everyone,

I applied for benefits on August 5th, and I have not received anything. Is there something wrong with my account?

**Comment by davevad93** [comment]
Hi @Ronak1Nambiar ,

Don't worry there's nothing wrong with your account, the waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ronak1Nambiar was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
